app/models/order.rb

```diff
@@ -2,6 +2,8 @@ class Order < ApplicationRecord
   has_many :order_products, dependent: :destroy
   has_many :products, through: :order_products
 
+  validates :email, :session_key, presence: true
```

Comment:
What is the reason to save `session_key` in Order? Seems like `cart_id` would be a better option.